Automattic/wp-calypso
# Command Palette: fix commands dont show in picker

Fixes #https://github.com/Automattic/dotcom-forge/issues/5088

## Proposed Changes

This pr aims to fix a bug, where sites don't appear for certain commands due to wrong capabilities

## Testing Instructions



1. Navigate to one of your site's dashboard. Ensure that you have at least one WoA site.
2. Open Command Palette. ( Using `Alt+K` or `Cmd+K` )
3. Search for `Install Plugin` command.
4. Ensure that there is at least one site listed.

**Before**

https://github.com/Automattic/wp-calypso/assets/497103/ff96d64e-f37f-478f-ba26-8583f4fc0add


 **After**

https://github.com/Automattic/wp-calypso/assets/497103/47b92704-d057-40ab-87b7-79e4d8f351c5

5. Ensure that the `SiteCapabilities` that changed, are the correct ones.
## Pre-merge Checklist
